app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Welcome, <%= @user.username %></h1>
+
+<ul>
+  <% Tweet.all.each do |tweet| %>
```

Comment:
I would change this to be `@tweets` and in the `tweets#index` part of the tweets_controller set the `@tweets = Tweet.all`